pkg/bath/straws.go

```diff
@@ -0,0 +1,194 @@
+package bath
+
+import (
+	"github.com/tonkeeper/tongo"
+	"github.com/tonkeeper/tongo/abi"
+	"github.com/tonkeeper/tongo/tlb"
+
+	"github.com/tonkeeper/opentonapi/internal/g"
+)
+
+type Straw func(bubble *Bubble) (success bool)
+
+var DefaultStraws = []Straw{
+	FindNFTTransfer,
```

Comment:
Probably, there is no need to export these functions:

```golang
var DefaultStraws = []Straw{
	findAndReplaceNftTransfer,
	findAndReplaceJettonTransfer,
}
```
and it would be great to stress somehow (naming?) that the functions modify the given bubble